src/command_modules/azure-cli-interactive/azclishell/__init__.py

```diff
@@ -3,4 +3,4 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
-__version__ = '0.3.3'
+__version__ = '0.3.5'
```

Comment:
Why the double bump?  @derekbekoe do we, as a matter of policy, wait to update the version until release?